docs/topics/getting-started-cd-osl.adoc

```diff
@@ -0,0 +1,97 @@
+= Continuous Delivery using {OpenShiftLocal}
+
+In addition to building on your local machine and pushing to a local or public cloud, you also have the ability to set up continuous delivery for a booster. Specifically you can use the {launcher} application to clone a new booster repository to your GitHub account and have it deployed to your {OpenShiftLocal}. 
+
+. link:{link-launcher-openshift-local-install-guide}#create-launcher-app[Install the {launcher}] application to your {OpenShiftLocal}.
+. Navigate to the {launcher} application on your {OpenShiftLocal} using your browser. You can also use the following command to open the {launcher} in your browser:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ minishift console
+----
+. Select _Prepare for Takeoff_.
+. Choose _Use {OpenShiftOnline} to build and deploy_.
+. Choose `REST API Level 0` and click _Next_.
+. Choose the {WildFlySwarm} runtime.
+. Enter an _OpenShift Project name_. You can also update any of the additional metadata fields or leave them as the default. Click _Next_.
+. Click the _Launch on OpenShift_ button.
++
+This will:
++
+** Create a new GitHub repository associated with your account containing the booster's code.
+** Create a pipeline to continuously deliver your booster's code.
+** The pipeline will then build and deploy your booster's code your {OpenShiftLocal}.
+
+. Click on the _See it here!_ link under _Creating your project on the OpenShift cloud_ to go to your booster's page in the OpenShift Web console.
+
+To view the status of your project:
+
+. Click _View Log_ in the pipeline at the top.
++
+NOTE: It may take a few minutes for the link to appear.
+
+. Click _Login with OpenShift_ at the login prompt for Jenkins. 
+. Click _Login with Red Hat_.
+. Ensure all the boxes are checked and click _Allow selected permissions_.
++
+This grants the OpenShift Web console the correct permissions and directs you to the Jenkins pod that is deploying your application.
+
+
+Once your deployment is complete, interact with your booster using the url for the booster application at the top of the page:
+
+IMPORTANT: Your booster application is not the same as your Jenkins pod, and it will only appear after the Jenkins pod deploys it. This may take a few minutes.
+
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ curl wfswarm-rest-http-my-osl-demo.192.168.64.2.nip.io/api/greeting/
+{"content":"Hello, World!"}
+$ curl wfswarm-rest-http-my-osl-demo.192.168.64.2.nip.io/api/greeting?name=Sarah
+{"content":"Hello, Sarah!"}
+----
+
+At this point, you have created a new booster repository in your GitHub account and deployed it to {OpenShiftLocal}. To push changes to this booster in {OpenShiftLocal}, you can push a new commit to the GitHub repository.
```

Comment:
This would require Ultrahook running?

Sorry, I'm already kinda lost -- I know we described Ultrahook somewhere, but after the big refactoring, I'm no longer sure if we still use that content.